pandas/core/indexes/interval.py

```diff
@@ -463,7 +463,13 @@ def is_unique(self):
         """
         Return True if the IntervalIndex contains unique elements, else False
         """
-        return self._multiindex.is_unique
+        left = self.values.left
```

Comment:
why isnt the answer just: ``self.left.is_unique & self.right.is_unique``

Reply:
That's a little too strict; you don't necessarily left or right uniqueness, only pairwise uniqueness, as you can have duplicate endpoints on one side as long as the other side also isn't the same, e.g. `[Interval(0, 1), Interval(0, 2), Interval(0, 3)]` should be unique.

Reply:
i think u can simply construct a numpy array and check that then via np.unique by first reshaping

Reply:
I have tried something below
```
arr = np.array(list(zip(self.values.left, self.values.right)))
np.unique(arr, axis=0)
```
But np.unique cannot filter out the duplicate np.nan

```
In [2]: arr = np.array([1, 3, 3, np.nan, np.nan])                               

In [3]: np.unique(arr)                                                          
Out[3]: array([ 1.,  3., nan, nan])
```

Reply:
use pd.unique